packages/dds/sequence/src/test/endpointInRangeIndex.spec.ts

```diff
@@ -0,0 +1,228 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { strict as assert } from "assert";
+import { makeRandom } from "@fluid-internal/stochastic-test-utils";
+import { Client } from "@fluidframework/merge-tree";
+import {
+	Interval,
+	IEndpointInRangeIndex,
+	createEndpointInRangeIndex,
+	intervalHelpers,
+} from "../intervalCollection";
+import {
+	assertPlainNumberIntervals,
+	createTestInterval,
+	generateRandomIntervals,
+} from "./intervalIndexUtils";
+
+class TestEndpointInRangeIndex implements IEndpointInRangeIndex<Interval> {
+	private readonly intervals: Interval[];
+
+	constructor() {
+		this.intervals = [];
+	}
+
+	add(interval: Interval) {
+		this.intervals.push(interval);
+	}
+
+	remove(interval: Interval) {
+		const index = this.intervals.findIndex((i) => i === interval);
+		if (index !== -1) {
+			this.intervals.splice(index, 1);
+		}
+	}
+
+	findIntervalsWithEndpointInRange(start: number, end: number): Interval[] {
+		return this.intervals.filter((interval) => interval.end >= start && interval.end <= end);
+	}
+}
+
+describe("findIntervalsWithEndpointInRange", () => {
+	let helpers;
```

Comment:
you can just define `helpers` and `compareFn` here, you don't need to reassign before each test.